RPI/Top/instances.fpp

```diff
@@ -23,12 +23,11 @@ module RPI {
   {
 
     phase Fpp.ToCpp.Phases.configObjects """
-    NATIVE_UINT_TYPE context[] = { RpiDemo::RG_CONTEXT_10Hz, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+    NATIVE_INT_TYPE context[] = { RpiDemo::RG_CONTEXT_10Hz, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
What is the reason for changing UINT to INT?

Reply:
I changed the type to INT to get rid of some compile errors when using them in loops.